### PR TITLE
Release 3.0.0-rc.6

### PR DESCRIPTION
**Pull request by blackbaud-conorwright**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "ng2-dragula": "1.5.0"
   },
   "devDependencies": {
-    "@blackbaud/skyux": "2.31.0",
+    "@blackbaud/skyux": "2.33.0",
```
Comment (Blackbaud-TrevorBurch) [inline package.json]: This isn't currently a real version @blackbaud-conorwright

Comment (blackbaud-conorwright) [reply package.json]: whoops, went too high ;p
Nice catch

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.6 (2018-12-06)
+
+- Fixed help button not showing when only using event listener. [#18](https://github.com/blackbaud/skyux-tiles/pull/18)
```
Comment (Blackbaud-SteveBrush) [inline CHANGELOG.md]: "Fixed (thing) to..." so, something like this?

"Fixed tile component's help button to display by default when consumer provides `helpClick` listener."

Comment (blackbaud-conorwright) [reply CHANGELOG.md]: nice, changed

**Pull request merged**